### DISCUSSION TITLE
Suggestion: Publish your add-on to the Blender Extensions Platform

### DISCUSSION THREAD

**Discussion by xan2622** [p2or/blender-loom#101] [Ideas]
Hi. 👋

As you may already know, add-ons and themes can now be uploaded to the [Blender Extensions Platform](https://extensions.blender.org/).

Related topics:
https://extensions.blender.org/about/
https://docs.blender.org/manual/en/latest/advanced/extensions/getting_started.html#how-to-create-extensions
https://code.blender.org/2024/05/extensions-platform-beta-release/
https://devtalk.blender.org/t/changes-to-add-on-and-themes-bundling-4-2-onwards/34593

I think it would be really convenient if your add-on was available on the Blender Extensions Platform. It would help Blender users discover and download your add-on more easily (it's also possible to search for (and install) extensions inside Blender).

Have you ever considered uploading your add-on on this new website?
Thanks in advance.

**Comment by p2or** [comment]
Hi @xan2622,

thanks for your suggestion. Good idea! Already on the to-do list and it's good to hear that people are interested. 

However, not high priority at the moment because I'm not really sure if it would be worth at this point of development as there is still a lot to do regarding the addon, e.g. playback is still missing some features that I would like to implement, plus publishing on the platform requires some code changes, and besides the platform still seems to be very experimental to me. I might be wrong, but I would expect some changes in the upcoming months so I'd personally wait at least one iteration to see whether the platform works as it was intended. Don't you think so?

Anyway, I think it would probably take 1 or 2 days of work. If you have any other arguments for immediate publication on the extension platform besides reaching a larger audience, please let me know, I'm open to all ideas.

Cheers,
Christian
